### DISCUSSION TITLE
GitHub Billing Support doesn't reply, even after 30+ days?

### DISCUSSION THREAD

**Discussion by s0urfruit** [community/community#121885] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hey there, 

GitHub seems to really struggle with Apple Subscriptions, and that's where I have my Pro plan billed. It worked flawlessly for months, up until early this year - where my Pro was 'cancelled' and was then reinstated with a coupon for a few months. 

The coupons expired, my Pages deployments are down, among other Pro-things -- now gone, and I don't feel comfortable adding my billing information to GitHub directly because I'm 99% sure I'll be double-billed... and support would take ages to reverse that too?

TL;DR, has the support staff been laid off? is it normal for 30+ day wait times? & when can I get the service I paid for back & credited?

The ticket # is #2706130.

Cheers!

**Comment by ebndev** [comment]
Hi @s0urfruit,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
